frontend/assets/theme/base.scss

```diff
@@ -14,3 +14,6 @@
 a {
   text-decoration: none;
 }
+body {
+  background-color: #272727 !important;
+}
```

Comment:
Add newline at the bottom. I also swear theres a variable for the `#272727` color? If there is, should use that.

Reply:
I couldn't find a variable, the colour only seems to be in the `.v-card, .v-sheet.v-list` css block.